tests/api2/test_300_nfs.py

```diff
@@ -123,17 +181,44 @@ def nfs_share(path, options=None):
         assert result.status_code == 200, result.text
 
 
+@contextlib.contextmanager
+def nfs_config(options=None):
+    '''
+    Use this to restore settings after the test
+    '''
```

Comment:
This could be correct, but IMO it might be worthwhile to add a comment explain why there are **two** different NFS config save/restore mechanisms, and _why_ they are called from where they are.

Reply:
done